playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -23,7 +23,9 @@ sudo git clone $EDX_THEME_REPO $dir_themes -b $THEME_BRANCH
 
 # todo:100627 this doesn't work on onebox installations (fullstack and devstack) which don't use oxa-tools-config
```

Comment:
consider rephrasing this todo to be something like: "onebox installations (fullstack and devstack) don't use oxa-tools-config so we need to test if these images are available before copying. i'd live the bug number as an fyi, but it no longer needs to be a todo

Reply:
If this change satisfies the TODO, it's simply better to remove it